libr/core/core.c

```diff
@@ -1276,7 +1277,11 @@ R_API int r_core_init(RCore *core) {
 
 	r_core_config_init (core);
 
+	ut64 diff, prev = r_sys_now();
 	r_core_loadlibs_init (core);
+	diff = r_sys_now() - prev;
+	sdb_num_add(DB, "r_core_loadlibs_init time", diff, 0);
```

Comment:
@radare not sure which file load time you are referring to, could you point out the code behind this load time ?
Also for displaying it in the `?T` command I assume I should add an entry in the `help_message[]` from cmd_help.c , but as most letters are taken what do you suggest ?
On the other hand I could add an entry in cmd_info , for something like `info t` which should display the loading times. What do you think ?